lxd/main_forkproxy.go

```diff
@@ -273,3 +321,78 @@ func getDestConn(connectAddr string) (net.Conn, error) {
 	addr := strings.Join(fields[1:], "")
 	return net.Dial(fields[0], addr)
 }
+
+func relay(src *net.UnixConn, dst *net.UnixConn) {
+	for {
+		dataBuf := make([]byte, 4096)
+		oobBuf := make([]byte, 4096)
```

Comment:
Hm, wouldn't it be safe and potentially more efficient if you'd move this out of the `for`-loop?

Reply:
Fixed